adding_to_the_RDynLib_library.qmd

```diff
@@ -29,17 +29,16 @@ register(SerialParam())
 first we load the XcmsExperiment ftms object
 
 ```{r}
-load(file = "C:/Users/amentag/Desktop/Rcodes/RDynLib_ftms/data/ftms_final2.RData")
+pth <- "C:/Users/amentag/Desktop/Rcodes/RDynLib_ftms/"
```

Comment:
I'm just defining this variable `pth` so that I can change the path to the files in a single place - as I have the files in a different folder on my local computer.